website/static/js/fangorn.js

```diff
@@ -1133,14 +1748,38 @@ tbOptions = {
         success : _fangornDropzoneSuccess,
         error : _fangornDropzoneError,
         dragover : _fangornDragOver,
-        addedfile : _fangornAddedFile
+        addedfile : _fangornAddedFile,
+        drop : _fangornDropzoneDrop
     },
     resolveRefreshIcon : function() {
         return m('i.fa.fa-refresh.fa-spin');
     },
     removeIcon : function(){
-        return m('i.fa.fa-times-circle');
-    }
+        return m.trust('&times;');
+    },
+    headerTemplate : _fangornToolbar,
+    // Not treebeard options, specific to Fangorn
+    iconState : _defaultIconState(),
+    defineToolbar : _fangornDefineToolbar,
+    onselectrow : function(row) {
+        console.log(row);
```

Comment:
I'm keeping that console.log until review is clean. It's immensely useful for development debugging.